### ISSUE TITLE
Can build

### ISSUE THREAD

**Issue by TGS-Bricks** [rgriebl/brickstore#731]
This software is just awesome.
It would be nice if I can make a new dokument with all the items listed in the "can build" list.

And

Let say I have a dokument with 5 set listed and another dokument with 50000 parts, it would be awesome if I can get a list with the parts that's missing for building these 5 sets.

TGS-Bricks

**Comment by TGS-Bricks** [comment]
Yes it's the automated feature I'm looking for, it would also be great if I can choose to get a "can build" list with lets say max 5 missing parts in the sets.

**Comment by StoneyBricks** [comment]
> > get a list with the parts that's missing
> 
> You can do this already. Unless you are saying you want it more automated.

How do you do that…manually?

**Comment by Leovaliant** [comment]
> How do you do that…manually?

Just have both bsx documents open (one with all your known parts & one with the set you wish to build)
From the all your parts bsx choose subtract items and select the bsx of the set you wish to build.  
Sort your all you parts bsx by qty and anything in negatives is the amount you're missing, If nothing is in the negatives, you can build it.  Keep in mind subtracting items is specific to the used/new condition.
